gnat2goto/driver/tree_walk.adb

```diff
@@ -1105,6 +1127,13 @@ package body Tree_Walk is
    -------------------
 
    function Do_Expression (N : Node_Id) return Irep is
+      function Create_Dummy_Irep return Irep;
```

Comment:
Maybe something like `Create_Stub_For_Missing_Implementation`? Wordy I know, but we need to take care to get these out of here again as quickly as possible.